Department-for-Transport-Disruptions/bods-integrated-data
# BODS-2186: Unsubscribe endpoint

To test:

- cd into `/src/functions/avl-unsubscriber` directory
- Run `pnpm run build:local`
- cd back to root directory
- Run `make setup` (this will run tf apply locally)
- Run `make create-avl-mock-data-producer` and go through steps to create a mock data producer in the local env
- Check DynamoDB table for the mock data producer and copy the subscription ID for that subscription, you can also check parameter store to see if username and password for that subscription ID has been created
- Run the command: `SUBSCRIPTION_ID=*YOUR_SUBSCRIPTION_ID* make invoke-local-avl-unsubscriber`
- Check DynamoDB table again to check if subscription status = TERMINATED, check parameter to see if username and password has been deleted
